test/performance/README.md

```diff
@@ -55,14 +56,18 @@ To run a benchmark once, and use the result from `mako-stub` for plotting:
    ko apply -f test/performance/benchmarks/broker-imc/200-broker-perf.yaml
    ```
 
+1. Wait until all the pods in namespace `perf-eventing` are completed.
+
 1. Retrieve results from mako-stub using the script in
-   [knative/pkg](https://github.com/knative/pkg/blob/master/test/mako/stub-sidecar/read_results.sh):
+   [knative/pkg](https://github.com/knative/pkg/blob/master/test/mako/stub-sidecar/read_results.sh)
+   where `pod_name` is the name of the aggreator pod:
 
    ```
    bash "$GOPATH/src/knative.dev/pkg/test/mako/stub-sidecar/read_results.sh" "$pod_name" perf-eventing ${mako_port:-10001} ${timeout:-120} ${retries:-100} ${retries_interval:-10} "$output_file"
```

Comment:
Since we are vendoring the mako stub, this path should be adjusted to the eventing vendor subdirectory.